server/src/main/java/org/opensearch/common/blobstore/stream/write/WritePriority.java

```diff
@@ -0,0 +1,17 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ */
+
+package org.opensearch.common.blobstore.stream.write;
+
+/**
+ * WritePriority for upload
+ */
+public enum WritePriority {
+    NORMAL,
+    HIGH
```

Comment:
do you have any PR already that has the logic on the priority?

Reply:
Yes, plugin PR linked in metadata will give you an idea of their role. This is currently used in providing stream processing bandwidth depending upon the priority.